book/src/keymap.md

```diff
@@ -53,9 +53,9 @@
 | `Ctrl-f`, `PageDown`  | Move page down                                     | `page_down`                 |
 | `Ctrl-u`              | Move half page up                                  | `half_page_up`              |
 | `Ctrl-d`              | Move half page down                                | `half_page_down`            |
-| `Ctrl-i`              | Jump forward on the jumplist                       | `jump_forward`              |
-| `Ctrl-o`              | Jump backward on the jumplist                      | `jump_backward`             |
-| `Ctrl-s`              | Save the current selection to the jumplist         | `save_selection`            |
+| `Ctrl-i`              | Jump forward on the jump list                      | `jump_forward`              |
+| `Ctrl-o`              | Jump backward on the jump list                     | `jump_backward`             |
+| `Ctrl-s`              | Save the current selection to the jump list        | `save_selection`            |
```

Comment:
For words like "jumplist" and "textobject" I think we can remove the space, these are kind of editor-specific language that I think is ok to use though it's not in the dictionary.

Reply:
Ok